gobblin-data-management/src/main/java/gobblin/data/management/copy/publisher/CopyDataPublisher.java

```diff
@@ -226,6 +230,7 @@ private void publishFileSet(CopyEntity.DatasetAndPartition datasetAndPartition,
 
     additionalMetadata.put(SlaEventKeys.SOURCE_URI, this.state.getProp(SlaEventKeys.SOURCE_URI));
     additionalMetadata.put(SlaEventKeys.DESTINATION_URI, this.state.getProp(SlaEventKeys.DESTINATION_URI));
+    additionalMetadata.put(SlaEventKeys.DATASET_OUTPUT_PATH, fileSetRoot.or(""));
```

Comment:
Perhaps it is better to make it clear the output path is unknown instead of leaving it empty.